spec/models/cma_case_spec.rb

```diff
@@ -117,6 +117,13 @@ def cma_case_content_item(n)
     it "returns all CMA Cases" do
       expect(described_class.all(page, per_page).results.length).to be(cma_cases.length)
     end
+
+    it "passes query parameter if supplied" do
+      expect(described_class.publishing_api).to receive(:get_content_items)
+        .with(hash_including(q: "foo"))
+        .and_return(double(to_ostruct: {}))
+      described_class.all(page, per_page, q: "foo")
+    end
```

Comment:
Do we also want to have a feature spec here to make sure the UI -> model part works (I'm sure it does, but...).